<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAdministratorDlg</class>
 <widget class="QDialog" name="CAdministratorDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QListWidget{
	background-color:#485464;
	padding-left:1px;
	padding-right:1px;
	outline:0px;
	border:0px;
}

QListWidget::item
{
	color:#cdcfd2;
	border-radius:5px;
	margin-top:1px;
	margin-bottom:1px;
    background-color:#cdcfd2;
}

QListWidget::Item:selected{
	color:#ffffff;
	border:3px solid #8dd1f4;
	background-color:#ffffff;
}

QListWidget::item:hover
{
	color:#ffffff;
    border:3px solid #8dd1f4;
	background-color:#ffffff;
   
}

QListWidget::item::selected:active
{ 
	color:#ffffff;
    border:3px solid #8dd1f4;
	background-color:#ffffff;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="UserTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="ProductPage">
      <attribute name="title">
       <string>Product</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QListWidget" name="ProductListWidget"/>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Product List</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="AddProductBtn">
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset resource="../images.qrc">
             <normaloff>:/images/add_device.png</normaloff>:/images/add_device.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>14</width>
             <height>14</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DeleteProductBtn">
           <property name="text">
            <string>Delete</string>
           </property>
           <property name="icon">
            <iconset resource="../images.qrc">
             <normaloff>:/images/delete_device.png</normaloff>:/images/delete_device.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>14</width>
             <height>14</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="UserPage">
      <attribute name="title">
       <string>User</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="6,4">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>User List</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="AddUserBtn">
             <property name="text">
              <string>Add</string>
             </property>
             <property name="icon">
              <iconset resource="../images.qrc">
               <normaloff>:/images/add_device.png</normaloff>:/images/add_device.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>14</width>
               <height>14</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="DeleteUserBtn">
             <property name="text">
              <string>Delete</string>
             </property>
             <property name="icon">
              <iconset resource="../images.qrc">
               <normaloff>:/images/delete_device.png</normaloff>:/images/delete_device.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>14</width>
               <height>14</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="UserListWidget"/>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Product List</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="UserAddProductBtn">
             <property name="text">
              <string>Add</string>
             </property>
             <property name="icon">
              <iconset resource="../images.qrc">
               <normaloff>:/images/add_mcu.png</normaloff>:/images/add_mcu.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>14</width>
               <height>14</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="UserDeleteProductBtn">
             <property name="text">
              <string>Delete</string>
             </property>
             <property name="icon">
              <iconset resource="../images.qrc">
               <normaloff>:/images/delete_mcu.png</normaloff>:/images/delete_mcu.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>14</width>
               <height>14</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="UserProductListWidget"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
